tool/tsh/resolve_default_addr.go

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2016-2022 Gravitational, Inc.
```

Comment:
I am quite sure we start the license notice year at the file creation year so I assume this should only say `2021`.

Reply:
Not sure _what_ was going on there. I'm almost impressed I managed to get it wrong in so many dimensions :-)